component/attr_cache/attr_cache.go

```diff
@@ -725,10 +670,21 @@ func (ac *AttrCache) RenameFile(options internal.RenameFileOptions) error {
 		ac.cacheLock.RLock()
 		defer ac.cacheLock.RUnlock()
 		// TODO: Can we just copy over the attributes from the source to the destination so we don't have to invalidate?
-		ac.deletePath(options.Src, renameTime)
-		ac.invalidatePath(options.Dst)
-	}
 
+		toBeDeleted, err := ac.cacheMap.get(options.Src)
+		if err != nil {
+			log.Err("could not find attr cache item due to following error: ", err)
```

Comment:
I think going forward, it would help to list the function name that's being referred to for PR comments since the line numbers are kind of useless for me when the comment becomes outdated.